code/game/objects/items/weapons/grenades/plastic.dm

```diff
@@ -105,10 +105,9 @@
 			else if(user.mind.gang_datum)
 				message_say = "[uppertext(user.mind.gang_datum.name)] RULES!"
 	user.say(message_say)
-	target = user
-	sleep(10)
-	prime()
-	user.gib(1, 1, 1)
+	explosion(user,0,2,0) //Cheap explosion imitation because putting prime() here causes runtimes
+	user.gib()
```

Comment:
You might be able to clone by putting in another body. And iirc you can still make a borg out of it.